### PR TITLE
Add my pictures and more features to make my portfolio more personal.

### PR DESCRIPTION
**Pull request by Cayle**

### CODE REVIEW

**Comment by rayliu0317** [comment]
So this pull request contains 4 commits, is that intended?

Also, the description indicates it's only a simple change to README, but actually it contains more involved changes. Consider sending pull requests for each separate commits.

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -13,16 +13,26 @@
...
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+
+
+/**
+ * Adds a random fun fact about me on each click.
+ */
+
+function randomFunFacts() {
+  const funFacts =
+      ['I"m a  Rising sophomore Computer Science major', 'This is my first internship', 'I was raised and high-schooled in Lagos, Nigeria', 'I speak three languages: English, and two native lanuages, Yoruba and Efik'];
```
Comment (rayliu0317) [inline portfolio/src/main/webapp/script.js:30]: This line is too long. Normally we recommend keeping each line short when possible for better readability. Can this array of strings split into different lines?

Comment (Cayle) [reply portfolio/src/main/webapp/script.js:30]: Okay, cool. I will do that. After making the change do I need to make a commit and send a new PR ?

**Comment by Cayle** [comment]
> Do you know there is a way for us to view your portfolio page? Screenshot maybe if it's not deployed anywhere yet.

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (rayliu0317) [reply portfolio/src/main/webapp/script.js:30]: You don't need to send a new PR. Just create new commit as you have been doing so far.

Comment (Cayle) [reply portfolio/src/main/webapp/script.js:30]: Okay, will do that. And for the screenshot, I can't find a way to include images in this comment box.

Comment (rayliu0317) [reply portfolio/src/main/webapp/script.js:30]: Does this help? https://help.github.com/en/github/managing-your-work-on-github/file-attachments-on-issues-and-pull-requests

Comment (Cayle) [reply portfolio/src/main/webapp/script.js:30]: Yes, it does. This is the current stage I am.
Screenshot 2020-05-27 at 3 54 38 PM

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -13,16 +13,29 @@
...
+
+/**
+ * Adds a random fun fact about me on each click.
+ */
+
+function randomFunFacts() {
+  const funFacts =
+      ['I"m a  Rising sophomore Computer Science major',
+       'This is my first internship', 
+      'I was raised and high-schooled in Lagos, Nigeria',
```
Comment (rayliu0317) [inline portfolio/src/main/webapp/script.js:32]: needs an extra space here to keep the indentation aligned.

**Comment by ljfranklin** [comment]
@Cayle I think the workflow we're going to try is always add both Ray and I as Reviewers on each PR, then one of us will Assign ourselves to do the actual review. I'll unassign myself and let you and Ray finish out the review.

**Comment by Cayle** [comment]
> @Cayle I think the workflow we're going to try is always add both Ray and I as Reviewers on each PR, then one of us will Assign ourselves to do the actual review. I'll unassign myself and let you and Ray finish out the review.

Okay, thanks. I will do that next time

**Pull request merged**